### PR TITLE
Find by point

### PR DESCRIPTION
**Pull request by benjaminleighton**

### CODE REVIEW

File: functions.py [hunk 1]
```diff
@@ -585,7 +588,40 @@
...
+    :type lat: float 
+    :param lon:
+    :type lon: float 
+    :param count:
+    :type count: int
+    :param offset:
+    :type offset: int
+    :return:
+    """
+    conn = await asyncpg.connect('postgresql://postgres:password@{}/mydb'.format(GEOBASE_ENDPOINT))
```
Comment (jyucsiro) [inline functions.py:603]: Is exposing user:pass a problem?

**Review by jyucsiro — APPROVED** [review]
Looks ok to me. Just a minor one about having credentials committed in code, but for purposes of progressing deployment, we can deal with that later if that's not too sensitive.

File: config.py [hunk 2]
```diff
@@ -4,7 +4,9 @@
 module = sys.modules[__name__]
 CONFIG = module.CONFIG = {}
 TRIPLESTORE_CACHE_URL = CONFIG["TRIPLESTORE_CACHE_URL"] = "http://db.loci.cat"
+TRIPLESTORE_CACHE_URL = CONFIG["TRIPLESTORE_CACHE_URL"] = "http://db.loci.cat"
```
Comment (ashleysommer) [inline config.py]: Is this an unintentionally duplicated line?

**Review by ashleysommer — APPROVED** [review]

File: functions.py (hunk 1 above)
Comment (benjaminleighton) [reply functions.py:603]: yes this is bad I will try to resolve in a future commit, the db will be behind a firewall and defaults should be changed and parameterized in anycase

**Pull request merged**